packages/ui-components/src/components/single-select-dropdown/single-select-dropdown.scss

```diff
@@ -23,4 +23,16 @@
 	kv-select {
 		--select-max-height: var(--dropdown-max-height);
 	}
+
+	.selection-clear {
+		@include kv-font-span-regular;
+		user-select: none;
+		color: kv-color('neutral-2');
+		cursor: pointer;
+	
+		&.disabled {
+			color: kv-color('neutral-5');
+			cursor: auto;
```

Comment:
Same as before, shouldn't it be 'not-allowed'?

Reply:
I'll check with the design team :)